src/main/java/core/basesyntax/FileReader.java

```diff
@@ -0,0 +1,261 @@
+ * Завершение работы программы
+ * <p>
+ * Если написать в консоль любой текст, который не является командой,
+ * и нажать enter, должно появиться сообщение с предложением сохранить текст
+ */
+
+class FileReader {
+    private static final String PATH_EXAMPLE = "D:\\\\java\\\\file-editor";
+    private static final String FILE_EXAMPLE = "file-editor";
+    private static List<String> commandList = new ArrayList<>(
+            Arrays.asList("create", "read", "info", "help", "help c", "exit"));
+    private String filePath = "";
+    private String fileName = "";
+
+    private String input() {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        try {
+            return reader.readLine();
+        } catch (IOException e) {
+            throw new RuntimeException("IOException", e);
+        }
+    }
+
+    private boolean createFile(String filePath, String fileName) {
+        if (!Files.exists(Paths.get(filePath))
+                && Files.isDirectory(Paths.get(filePath))) {
+            System.out.println("=== Path is not exist ===");
+            return false;
+        } else if (Files.exists(Paths.get(filePath + "\\" + fileName))) {
+            System.out.println("=== Would you rewrite file ===\nY / N");
+            while (true) {
+                if (input().equals("N")) {
+                    return false;
+                } else if (input().equals("Y")) {
+                    try {
+                        Files.createFile(Paths.get(filePath + "\\" + fileName));
+                        break;
+                    } catch (IOException e) {
+                        throw new RuntimeException(e);
+                    }
+                } else {
+                    System.out.println("=== Incorrect input ===");
+                }
+            }
+        } else {
+            try {
+                Files.createFile(Paths.get(filePath + "\\" + fileName));
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+        }
+        return true;
+    }
+
+    private boolean readFile(String filePath, String fileName) {
+        if (!Files.exists(Paths.get(filePath))) {
+            System.out.println("=== Path is not exist ===");
+            return false;
+        } else if (!Files.exists(Paths.get(filePath + "\\" + fileName))) {
+            System.out.println("=== File is not exist ===");
+            return false;
+        } else {
+            try {
+                if (Files.readString(Paths.get(filePath + "\\" + fileName)).equals("")) {
+                    System.out.println("=== File is empty ===");
+                } else {
+                    System.out.println(Files.readString(Paths.get(filePath + "\\" + fileName)));
+                }
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+        }
+        return true;
+    }
+
+    private boolean info(String filePath, String fileName) {
```

Comment:
```suggestion
    private boolean getInfo(String filePath, String fileName) {
```
https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names